packages/medusa/src/api/routes/admin/draft-orders/create-draft-order.ts

```diff
@@ -120,14 +121,26 @@ import { validator } from "../../../../utils/validator"
 export default async (req, res) => {
   const validated = await validator(AdminPostDraftOrdersReq, req.body)
 
-  const value = transformIdableFields(validated, [
-    "shipping_address",
-    "billing_address",
-  ])
+  const { shipping_address, billing_address, ...rest } = validated
+
+  const draftOrderDataToUpdate: DraftOrderCreateProps = { ...rest }
```

Comment:
todo: Rename data passed to the DraftOrderService
```suggestion
  const draftOrderDataToCreate: DraftOrderCreateProps = { ...rest }
  ...
```
-------------
suggestion: Move it to the service layer and simplify it similar to how we do in the CartService:
```
const shippingAddress = data.shipping_address_id ?? data.shipping_address
const billingAddress = data.billing_address_id ?? data.billing_address
```

Reply:
Regarding the suggestion, I can't change how it's passed right now with out changing up the cart props type which I don't think should be part of this PR. 😄

Reply:
I suggested migrating it to behave exactly like the `update-cart.ts` flow. I stole the above snippet from the `CartService.update`. Am I missing something?